libpod/container_internal.go

```diff
@@ -1332,9 +1332,9 @@ func (c *Container) mount() (string, error) {
 }
 
 // unmount unmounts the container's root filesystem
-func (c *Container) unmount() error {
+func (c *Container) unmount(force bool) error {
 	// Also unmount storage
```

Comment:
Force should not be allowed if the container if in ContainerStateRunning or ContainerStatePaused, or if there are one or more active exec sessions - we don't want people force-unmounting a running container.

Reply:
Good catch - nice to know we were thinking of these kind of problems when we wrote it originally.